app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,31 @@
+class CommentsController < ApplicationController
+  # before_action :set_post, only: [:create, :destroy]
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.build(comment_params)
+    @comment.author_id = current_user.id
+
+    if @comment.save
+      redirect_to user_post_path(@post.user, @post), notice: 'Comment was successfully added.'
+    else
+      redirect_to user_post_path(@post.user, @post), alert: 'Failed to add the comment.'
+    end
+  end
```

Comment:
- Hi, Can you add  `new` action to the comments controller as you did for `create` action? 
- **Hint**: 
```ruby
def new
    @comment = Comment.new
  end
```
- Also, create a form that accepts input for creating comments. I would advise you to follow this directory `views/comments/new.html.erb` as you did with post
- NB: The comment above resolved from the project requirements